src/tile.cpp

```diff
@@ -1612,15 +1600,15 @@ Item* Tile::getUseItem(int32_t index) const
 	}
 
 	// try getting thing by index
-	if (Thing* thing = getThing(index)) {
-		Item* thingItem = thing->getItem();
+	if (auto thing = getThing(index)) {
+		auto thingItem = thing->getItem();
 		if (thingItem) {
 			return thingItem;
 		}
 	}
 
 	// try getting top movable item
-	Item* topDownItem = getTopDownItem();
+	auto topDownItem = getTopDownItem();
```

Comment:
Could this be `const auto`
```suggestion
	const auto topDownItem = getTopDownItem();
```